README.rst

```diff
@@ -84,13 +84,18 @@ is the directory with custom plugins of oh-my-zsh `(read more) <https://github.c
 
     git clone "https://github.com/MichaelAquilina/zsh-autoswitch-virtualenv.git" "$ZSH_CUSTOM/plugins/autoswitch_virtualenv"
 
-
 Then add this line to your ``.zshrc``
 
 ::
 
     plugins=(autoswitch_virtualenv $plugins)
 
+Manual Installation
```

Comment:
Could you mark this as bold to make it obvious its a subtitle?

Also, in a separate line below it could you add something like

"Source the plugin shell script in your `~/.zshrc`profile."

Reply:
Done